internal/transport/rpc.go

```
@@ -0,0 +1,64 @@
+	err = rpcClient.Call(rpcServiceName, rpcRequest, rpcResponse)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func RegisterServices(services []interface{}) {
+	logger.Info(common.COMPONENT_COMMON, "Registering Services...")
+	for _, service := range services {
+		rpc.Register(service)
```

golangci-lint (1.64.8)

Error return value of `rpc.Register` is not checked

(errcheck)